Formula/fuego-firestore.rb

```diff
@@ -0,0 +1,18 @@
+class FuegoFirestore < Formula
+  desc "Fuego is a command-line client for the firestore database"
+  homepage "https://github.com/sgarciac/fuego"
+  url "https://github.com/sgarciac/fuego/archive/refs/tags/0.31.1.tar.gz"
+  sha256 "4657ba2854a28d3cf58c0818c2088b650979cfd3cb5c1dd3be4669b88b1cd870"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "#{bin}/fuego"
+  end
+
+  test do
+    version_output = shell_output("#{bin}/fuego --version")
+    assert_match "Fuego version #{version}", version_output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thank you for the feedback, test updated.